docs/buildah-push.md

```diff
@@ -40,6 +40,14 @@ Image stored in local container/storage
 
 ## OPTIONS
 
+**--cert-dir** *path*
+
+Use certificates at *path* (*.crt, *.cert, *.key) to connect to the registry
+
+**--creds** *creds*
+
+The username and password to use to authenticate with the registry if required.
```

Comment:
Format username[:password]

Reply:
yes, same as commit.  Ty for picking this one up, I missed that.